### ISSUE TITLE
Black GUI

### ISSUE THREAD

**Issue by Akull9** [jakimfett/Werkbench#14]
With a clean install with Forge 1492 and the latest release of Werkbench, 1.2.56:
![2015-08-13_13 58 18](https://cloud.githubusercontent.com/assets/6075254/9261549/7dbf694a-41c3-11e5-8204-c27628f2d19b.png)
![2015-08-13_13 58 25](https://cloud.githubusercontent.com/assets/6075254/9261548/7db8d166-41c3-11e5-8c02-2b7c857f7220.png)

I've experienced this issue with EasyCrafting and Steve's Production table as well...

**Comment by jakimfett** [comment]
Odd, and that's with no other mods installed at all? Just Werkbench, nothing else?

**Comment by Akull9** [comment]
Yeah; not sure what's causing it. On the Production Table issue, "add getColor can fix this issue" was posted by someone. Maybe that helps?

**Comment by Akull9** [comment]
https://github.com/loldudester/EasyCrafting/commit/e0a3cab3173ab65f658dbe1cbcec6c2855475b99

The issue was fixed in EasyCrafting, could you possibly do the same to Werkbench?
